src/themes/oneui/mixins/text.scss

```diff
@@ -21,6 +21,6 @@ $heading-font-sizes: $h1-font-size, $h2-font-size, $h3-font-size, $h4-font-size;
 @mixin heading($n: 1) {
     font-family: $heading-font-family;
     font-size: nth($heading-font-sizes, $n);
-    color: $color-secondary;
+    color: $color-primary;
```

Comment:
this looks like it should have been broken already. Is this function used?

Reply:
looks like we dont use it, as other functions from this file...